myocamlbuild.ml

```diff
@@ -941,7 +941,7 @@ dispatch
                         "gettext" 
                         (BaseEnvLight.load
                            ~allow_empty:true
-                           ~filename:(Lazy.force MyOCamlbuildBase.env_filename)
+                           ~filename:(MyOCamlbuildBase.env_filename)
```

Comment:
On Sun, 4 Oct 2015 11:10:52 -0700, gerdstolpmann wrote:
> 
> In myocamlbuild.ml:
> 
> > @@ -941,7 +941,7 @@ dispatch
> >                          "gettext"
> >                          (BaseEnvLight.load
> >                             ~allow_empty:true
> > -                           ~filename:(Lazy.force MyOCamlbuildBase.env_filename)
> > +                           ~filename:(MyOCamlbuildBase.env_filename)
> 
> Had to do this to compile oasis. No idea why. If it is wrong, I can revert this change.

This is because MyOCamlbuildBase.env_filename is different in the development version of oasis.  You probably were trying to compile it with the oasis executable from an earlier version?